threadpool/core/src/main/java/cn/hippo4j/core/plugin/impl/TaskTimeRecordPlugin.java

```diff
@@ -147,13 +169,8 @@ private Timer getTimerForCurrentThread() {
      * copy from {@link HashMap#tableSizeFor}
      */
     static int tableSizeFor(int cap) {
```

Comment:
Please add unit test for this method

Reply:
This is the implementation of the `HashMap#tableSizeFor` method in `jdk 11`. If needed, I will provide additional unit test cases.

Reply:
Need to provide, the unit test needs to write the result comparison of two pieces of code.